react-frontend/src/components/Navigation/NavLink/NavLink.css

```diff
@@ -0,0 +1,20 @@
+.nav-link {
+  flex: 1;
+  text-align: center;
+  padding: 10px 0;
+  text-decoration: none;
+  color: #333;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  margin: 0 2.5%; /* This ensures the 25% space distribution */
```

Comment:
This comment seems to be misleading. Do we really need it?

Reply:
Agree. Refactored in [a84bbb1](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/a84bbb17628b10dc091f5c23da6a0e28ca6a073a)